### DISCUSSION TITLE
Retaining chapters upon export?

### DISCUSSION THREAD

**Discussion by kanishknishar** [mifi/lossless-cut#2022] [Q&A]
I am remuxing classical DVDs so that a single video file containing multiple works is split off into separate files. At the moment, when I export by segment, the video file merges all segments and I retain movement info which would be really nice to have. Is there a way to do this?

Side-question: Is there a way to rename chapters en masse quickly? I have several concerts and each video could have 15-20 songs so it would be really nice if I could copy and paste the list of songs directly a la foobar mass-renaming of titles.

**Comment by mifi** [comment]
hi. not sure exactly what you do, could you provide a step by step instruction of what you do, what you see, and then what you'd instead like to see?
